packages/core/helper-plugin/lib/src/components/SearchURLQuery/index.js

```diff
@@ -38,7 +38,11 @@ const SearchURLQuery = ({ label, placeholder, trackedEvent }) => {
 
     if (value) {
       if (trackedEvent) {
-        trackUsage(trackedEvent);
+        if (trackedEventDetails) {
+          trackUsage(trackedEvent, trackedEventDetails);
+        } else {
+          trackUsage(trackedEvent);
+        }
```

Comment:
I'd suggest to simplify this spaghetti to:

```suggestion
        trackUsage(trackedEvent, trackedEventDetails);
```

By default `trackedEventDetails` is undefined, which is perfectly fine to pass it into `trackUsage` I think?

Reply:
I think it is fine, I'll do a test first!